cli/flox-rust-sdk/src/providers/publish.rs

```diff
@@ -42,8 +77,10 @@ pub struct CheckedEnvironmentMetadata {
 
 /// Ensures that the required metadata for publishing is consistent from the build process
 #[allow(clippy::manual_non_exhaustive)]
+#[derive(Debug)]
 pub struct CheckedBuildMetadata {
```

Comment:
suggestion: more derives
```suggestion
#[derive(Debug, Clone, PartialEq, Eq)]  
```